certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -866,15 +959,13 @@ PROCEDURE INITIALIZATIONS
  LFNUMBER=0
  ERRORCODE=0
  USEROPTIONSMAXUSED=0
-
- PARAMUSEROPTIONS="1,2,3,4,5,6"
  PARAMLIMITCHANGES=TRUE
  PARAMINELIGIBLEACCTS="NONE"
  PARAMWARNINGSACCT="NONE"
  PARAMWARNINGSSHARE="NONE"
  PARAMDELETEXFERS=FALSE
 
```

Comment:
I recommend blanking out the new PARAMSHARETYPESBYOPTION(6,9999) Array in this INITIALIZATIONS Procedure, as right now it is not getting initialized to blank. Or you can initialize Options 1-6 in PARAMSHARETYPESBYOPTION to ALL, if that is the default.

Reply:
Really not necessary since the array will pull all the true/false values from the results of the LISTEXPAND array.  That being said, I added it in since it wouldn't hurt and would also be more consistent with other code